opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1230,7 +1237,7 @@ namespace Opm {
                                           DeferredLogger& local_deferredLogger)
     {
         OPM_TIMEFUNCTION();
-        auto [well_group_control_changed, more_network_update] =
+        auto [well_group_control_changed, inner_network_not_balanced] =
```

Comment:
If we change the name `more_network_update` to `inner_network_not_balanced`, it might be good to update the variable name in the function `updateWellControls`  also to avoid naming confusion.